src/test/java/org/broadinstitute/hellbender/engine/dataflow/RefAPISourceUnitTest.java

```diff
@@ -112,6 +113,28 @@ public void testReferenceSourceQueryWithNullInterval() {
     }
 
 
+    @Test(groups = "cloud")
+    public void testReferenceSourceMultiPageQuery() {
+        final int mio = 1_000_000;
```

Comment:
this is cool, I didn't know you could separate with underscores like that in constants